Sources/SnowManLogic/ViewLogic/MockView.swift

```diff
@@ -0,0 +1,40 @@
+public class MockView: Viewable {
+	var ui: Receivable 
+	var guess: String!
```

Comment:
Hah I thought the better was was an empty `String`. I know that is not the correct answer. Standby for this.